src/screens/AddOrEditPost/selectImage.ts

```diff
@@ -0,0 +1,46 @@
+import { PermissionsAndroid, Platform } from "react-native";
+import ImagePicker from "react-native-image-crop-picker";
+import ImageResizer from "@bam.tech/react-native-image-resizer";
+
+const resizeImage = (path: string, setImageURL: (url: string) => void) => {
+  ImageResizer.createResizedImage(path, 1000, 1000, "JPEG", 10)
+    .then((response) => {
+      let pathAfterResize =
+        Platform.OS === "android" ? `file://${response.path}` : response.path;
+      setImageURL(pathAfterResize ?? "");
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
Borde vi visa något fel för användaren om den misslyckas ändra storleken? Jag antar det stoppar hela processen med att ladda upp en bild?

Reply:
men vad kan användate göra med detta? Om det funkar inte då behöver vi fixa det annars användare kan inte göra nåt med detta :)

Reply:
now it show alert with text    "Något gick fel. Starta om appen och prova igen. Om det inte hjälper, kontakta din konsultchef."